docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,30 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule-To-Start Timeout is the maximum amount of time that is allowed from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (that is, placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (that is, picks up from the Task Queue) that Activity Task.
+In other words, it's a limit for how long an Activity Task can be enqueued.
+The moment that the Task is picked by the Worker from the task queue is considered to be the start of the Activity Task for the Schedule To Start timeout and associated metrics to avoid clock difference between the Server and the Worker.
```

Comment:
```suggestion
The moment that the Task is picked by the Worker from the Task Queue is considered to be the start of the Activity Task for the purposes of the Schedule-To-Start timeout and associated metrics to avoid clock difference between the Server and the Worker.
```